### PR TITLE
Consistent use of #pragam once in emscripten headers

### PR DESCRIPTION
**Pull request by sbc100**

### CODE REVIEW

File: system/include/emscripten/bind.h [hunk 1]
```diff
@@ -9,7 +9,7 @@
 
 #if __cplusplus < 201103L
 #error Including <emscripten/bind.h> requires building with -std=c++11 or newer!
-#else
```
Comment (jgravelle-google) [inline system/include/emscripten/bind.h]: Nice

File: system/include/emscripten/websocket.h [hunk 2]
```diff
@@ -1,3 +1,10 @@
+/*
+ * Copyright 2018 The Emscripten Authors.  All rights reserved.
```
Comment (jgravelle-google) [inline system/include/emscripten/websocket.h]: 2019?

Comment (sbc100) [reply system/include/emscripten/websocket.h]: I'm using the year of file creation here.

**Comment by juj** [comment]
lgtm, thanks!

**Pull request merged**